modules/source/kafka/config/kafka.xml

```diff
@@ -68,23 +58,42 @@
 	</bean>
 
 	<bean id="kafkaInboundChannelAdapter"
```

Comment:
How about to change to the namespace component - `<int-kafka:message-driven-channel-adapter>` ?